modules/ipi-install-creating-a-disconnected-registry.adoc

```diff
@@ -7,7 +7,7 @@
 
 = Creating a disconnected registry (optional)
 
-In some cases, you might want to install an Openshift KNI cluster using a local copy of the installation registry. This could be for enhancing network efficiency because the cluster nodes are on a network that does not have access to the internet.
+In some cases, you may want to install an OpenShift KNI cluster using a local copy of the installation registry. This could be for enhancing network efficiency because the cluster nodes are on a network that does not have access to the internet.
```

Comment:
```suggestion
In some cases, you might want to install an OpenShift KNI cluster using a local copy of the installation registry. This could be for enhancing network efficiency because the cluster nodes are on a network that does not have access to the internet.
```
It's always better to use "might" instead of "may" because it doesn't translate well and implies that the software is granting permission.